app/cdap/text/text-en.yaml

```diff
@@ -3436,4 +3436,18 @@ features:
       gridTable:
         title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+    GridPage:
+      directivePanel:
+        moreInfoOnDirective: More info to this directive
```

Comment:
more info about this directive ?

Reply:
Fixed replaced to with about in the text